internal/config/config.go

```diff
@@ -89,6 +89,65 @@ type Config struct {
 
 // }
 
+// defaultConfig
+func defaultConfig() Config {
+	config := Config{}
+
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return applyEnvironmentConfiguration(config)
+	}
+
+	tcProp := filepath.Join(home, ".testcontainers.properties")
+	// Init from a file, ignore if it doesn't exist, which is the case for most users.
+	// The properties library will return the default values for the struct.
+	props, err := properties.LoadFiles([]string{tcProp}, properties.UTF8, true)
```

Comment:
bug: looks like this could fail due to bad config and not inform the user, making them think their config is being used but it isn't. Explicitly check of `fs.ErrNotExist`?

Reply:
Do you think we can move to a separate issue for bootstrapping the config in a more consistent manner?

Reply:
That depends, is this moving code or introducing it, if the latter I would suggest this is fixed in this PR so we don't introduce new issues, all be it temporarily.

Reply:
The magiconair library already checks that if the ignoreMissing param is set to true: https://github.com/magiconair/properties/blob/c9a06e8f8f0164e4e16c0d5c4793cbed4ac90264/load.go#L93-L102

```go 
// LoadFile reads a file into a Properties struct.
// If IgnoreMissing is true then a missing file will not be
// reported as error.
func (l *Loader) LoadFile(filename string) (*Properties, error) {
	data, err := ioutil.ReadFile(filename)
	if err != nil {
		if l.IgnoreMissing && os.IsNotExist(err) {
			LogPrintf("properties: %s not found. skipping", filename)
			return NewProperties(), nil
		}
		return nil, err
	}
	return l.loadBytes(data, l.Encoding)
}
```